demo/chatbot_simple_demo/run.py

```diff
@@ -1,17 +1,25 @@
 import gradio as gr
 import random
-
-def respond(chat_history, message):
-  response = random.choice(["Yes", "No"])
-  return chat_history + [[message, response]]
+import time
 
 with gr.Blocks() as demo:
     chatbot = gr.Chatbot()
     msg = gr.Textbox()
     clear = gr.Button("Clear")
 
-    msg.submit(respond, [chatbot, msg], chatbot)
+    def user_message(message, history):
+        return history + [[message, None]]
```

Comment:
Let's make the textbox clear on submit as well?
```suggestion
    def user_message(message, history):
        return "", history + [[message, None]]
```